tests/test_spark.py

```diff
@@ -318,6 +318,53 @@ def test_secrets():
         LOGGER.warn("Error when deleting secret, {}".format(r.content))
 
 
+# Skip DC/OS < 1.10, because it doesn't have support for file-based secrets.
+@pytest.mark.skip("Enable when SPARK-22131 is merged and released in DC/OS Spark")
```

Comment:
I assume you've tested this locally. How about we test with a specific docker image (that has the patch, built master) instead of the universe package. I don't like the idea of merging "skipped tests"